### PR TITLE
Rename scripts to prepare for the "tag-details"->"remote" and "stats"->"local" transitions (including converting "tag-details" to have the ability to run as a long-running daemon so we can eke more efficiency out of our shared cache and easily generate multiple files)

### PR DESCRIPTION
**Pull request by tianon**
cc @yosifkit

### CODE REVIEW

File: .remote.pl [hunk 1]
```diff
@@ -383,7 +381,7 @@ sub cmd_to_dockerfile {
 
 		my $str = $cmd->[0];
 		my @prefix = ();
-		if ($str =~ s!^[|]\d+ (.*?) $shellRegex!$2!) {
+		if ($str =~ s!^[|]\d+ (.*?) ($shellRegex)!$2!) {
```
Comment (tianon) [inline .remote.pl]: minor bugfix

File: .remote.pl [hunk 2]
```diff
@@ -130,7 +128,7 @@ sub get_manifest {
 	return () if $manifestTx->res->code == 404; # tag doesn't exist
 	die "failed to get manifest for $image" unless $manifestTx->success;
 	return (
-		$manifestTx->res->headers->header('Docker-Content-Digest'),
+		$digests{$image} = $manifestTx->res->headers->header('Docker-Content-Digest'),
```
Comment (tianon) [inline .remote.pl]: minor bugfix (only seen if requesting the same image twice :innocent:)

File: update-remote.sh [hunk 3]
```diff
@@ -39,7 +39,7 @@ for repo in "${repos[@]}"; do
 			href="#${href,,}"
 			echo $'-\t[`'"$tag"'`]('"$href"')'
 		done
-		docker run -i --rm repo-info:tag-details "${tags[@]}"
+		docker run -i --rm repo-info:remote -- "${tags[@]}"
```
Comment (tianon) [inline update-remote.sh]: I didn't convert this file to use a single `docker run` instance yet since this PR was already big enough as-is. :+1:

**Comment by yosifkit** [comment]
LGTM

**Pull request merged**